packages/dds/tree/src/test/testTrees.ts

```diff
@@ -145,25 +132,27 @@ const builder = new SchemaBuilderBase(FieldKinds.required, {
 	scope: "test",
 	libraries: [leaf.library],
 });
-export const minimal = builder.object("minimal", {});
-export const hasMinimalValueField = builder.object("hasMinimalValueField", {
-	field: minimal,
-});
-export const hasNumericValueField = builder.object("hasNumericValueField", {
-	field: leaf.number,
-});
-export const hasPolymorphicValueField = builder.object("hasPolymorphicValueField", {
-	field: [leaf.number, minimal],
-});
+const factory = new SchemaFactory("test");
+const minimal = builder.object("minimal", {});
```

Comment:
is this still needed?

Reply:
I think it's just the "anyFields-full" tree that uses it so could we replace it with something else?

Reply:
I wanted to avoid making any actual schema/tree changes as part of this so I kept it, but yes in a future change we can probably remove this (along with the tests using "any")